manim/mobject/three_dimensions.py

```diff
@@ -732,6 +838,86 @@ def get_start(self):
     def get_end(self):
         return self.end
 
+    @classmethod
+    def perp_to_line(
+        cls, point: Sequence[float], line: "Line3D", length: float = None, **kwargs
+    ) -> "Line3D":
+        """Returns a line passing through a point and
+        perpendicular to another line.
+
+        Example
+        -------
+
+        .. manim:: PerpendicularLineExample
+            :save_last_frame:
+
+            class PerpendicularLineExample(ThreeDScene):
+                def construct(self):
+                    self.set_camera_orientation(60 * DEGREES, -135 * DEGREES)
+                    ax = ThreeDAxes()
+                    line1 = Line3D(ax.c2p(4, 0, 3), ax.c2p(0, -3, 0), color=RED)
+                    dot = Dot3D(ax.c2p(-3, 2, 0), color=BLUE)
+                    line2 = Line3D.perp_to_line(dot.get_center(), line1, length=7, color=YELLOW)
+                    self.add(ax, line1, dot, line2)
+
+        Parameters
+        ----------
+        point: Sequence[float]
+            A numpy array of the point at which the line passes
+        line: :class:`Line3D`
+            The line to be perpendicular to.
+        length: float
+            The length of the line.
+        """
+        u = point - line.get_center()
+        v = line.vect
+        length = length or line.length
+        intersection = projected_vector(v, u) + line.get_center()
+        direction = point - intersection
+        direction /= np.linalg.norm(direction)
+        return Line3D(
+            intersection + direction * length / 2,
+            intersection - direction * length / 2,
+            **kwargs,
+        )
+
+    @classmethod
+    def parallel_to_line(
+        cls, point: Sequence[float], line: "Line3D", length: float = None, **kwargs
+    ) -> "Line3D":
```

Comment:
The default of `None` breaks the code here, it's not implemented as it was in `perp_to_line`
```suggestion
    @classmethod
    def parallel_to_line(
        cls, point: Sequence[float], line: Line3D, length: float = None, **kwargs
    ) -> Line3D:
```

Reply:
I can fix the length attribute, but why should the quotation marks be dropped? it leaves a undefined error if it's dropped